src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/global.yml

```diff
@@ -676,10 +676,10 @@ Resources:
           phases:
             install:
               runtime-versions:
-                python: 3.9
-                nodejs: 14
+                python: 3.11
+                nodejs: 18
               commands:
-                - npm install cdk@1.188 -g -y --quiet --no-progress
+                - npm install cdk@2.79 -g -y --quiet --no-progress
```

Comment:
Is this package even needed here? 

I was trying to re-integrate the contents of this pull request into our testing ADF environment, but it failed due to the package cdk not found in version 2.79. For some reason, it seems that there's no 2.79 version published (as of 2023-05-15, see here: https://www.npmjs.com/package/cdk?activeTab=versions) but there's a 2.79 version of aws-cdk and the other cdk libraries: https://www.npmjs.com/package/aws-cdk-lib?activeTab=versions. 

Could we perhaps remove it if we don't need it (or if we need it, we might need to pick another package, should it be aws-cdk)?

Reply:
I think you are correct, we need to use `aws-cdk`. The `cdk` package only links to `aws-cdk/bin/cdk` anyway.

See: https://github.com/aws/aws-cdk/blob/main/packages/cdk/bin/cdk

Reply:
I tried to understand what we actually needed the npm cdk package for here in the first place though? 

We are running some python scripts in this codebuild projects buildspec definition, but never calling the cdk cli or any JavaScript code as far as I can see, so I'm not sure I understand the need for the npm cdk package (or aws-cdk) either way in the first place? Or are we calling some cdk cli indirectly from the python scripts?

Reply:
That is probably a left over from #424 and we no longer need `cdk` here indeed. @StewartW is that correct?

Reply:
That's correct. 👍